e2e/js_run_devserver/WORKSPACE.bzlmod

```diff
@@ -0,0 +1,15 @@
+# The presence of this file causes WORKSPACE to be ignored when bzlmod is enabled.
+# See https://docs.google.com/document/d/1JtXIVnXyFZ4bmbiBCr5gsTH4-opZAFf5DMMb-54kES0/edit#heading=h.y054fjub9max
+
+load("@bazel_tools//tools/build_defs/repo:http.bzl", "http_archive")
+
+http_archive(
+    name = "com_github_ash2k_bazel_tools",
```

Comment:
Let's file these on Bazel-central-registry as we encounter them. I'd like to ask Google to fund bzlmod rollout but we have to show them a "percent missing" or something

Reply:
https://github.com/bazelbuild/bazel-central-registry/issues/419